// Error: break outside of a loop
package main

func main () {
	break
}